ext/standard/basic_functions.stub.php

```diff
@@ -792,7 +806,10 @@ function fclose($stream): bool {}
 /** @param resource $stream */
 function feof($stream): bool {}
 
-/** @param resource $stream */
+/**
+ * @param resource $stream
+ * @refcount 1
+ */
```

Comment:
Noticed that this doesn't really make sense ... I've improved the implementation in https://github.com/php/php-src/commit/6e20f0f3a2ad59cb1261eafd82bf32da8082b975.